common/sgx/quote.c

```diff
@@ -529,6 +542,208 @@ oe_result_t oe_verify_quote_with_sgx_endorsements(
     }
 
     oe_datetime_log("Validation datetime: ", &validation_time);
+
+    // Convert validation time to time_t
```

Comment:
Suggest adding a comment highlighting this is insecure, since it relies on untrusted time information from the host.
```suggestion
    // Convert validation time to time_t
    // NOTE: This is insecure, since it relies on untrusted time information from the host.
    // However, it's better than nothing since it can still defend against some remote attacks.
```

Reply:
In fact, it depends on enclave-side verification or host-side verification. 
In enclave-side verification, the time parsed from verification collaterals are protected by QvE, and OE SDK uses function `oe_verify_qve_report_and_identity()` to verify the QvE returned info, including the time here. 

So I would not add `not secure` in comments.